src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java

```diff
@@ -85,6 +85,10 @@ public static Field makeField(SQLExpr expr, String alias, String tableAlias) thr
             methodParameters.add(new KVValue(alias));
             methodParameters.add(new KVValue(scriptCode));
             return new MethodField("script", methodParameters, null, alias);
+        } else if (expr instanceof SQLIntegerExpr) {
```

Comment:
is it only for integers? how about `SELECT 1.0 FROM myData`?

Reply:
Good point! Add the support for numeric literals.